pandas/core/tools/datetimes.py

```diff
@@ -898,7 +898,7 @@ def to_datetime(
     Passing ``errors='coerce'`` will force an out-of-bounds date to :const:`NaT`,
     in addition to forcing non-dates (or non-parseable dates) to :const:`NaT`.
 
-    >>> pd.to_datetime("13000101", format="%Y%m%d", errors="coerce")
+    >>> pd.to_datetime("invalid for Ymd", format="%Y%m%d", errors="coerce")
```

Comment:
So this now infers Jan 01, 1300 with a second resolution right?

Reply:
correct